AsyncDisplayKit/Details/_ASDisplayViewAccessiblity.mm

```diff
@@ -0,0 +1,153 @@
+/* Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "_ASDisplayViewAccessiblity.h"
+#import "_ASDisplayView.h"
+#import "ASDisplayNode+FrameworkPrivate.h"
+
+#import <objc/runtime.h>
+#import <queue>
+
+
+#pragma mark - UIAccessibilityElement
+
+static const char *ASDisplayNodeAssociatedNodeKey = "ASAssociatedNode";
+
+@implementation UIAccessibilityElement (_ASDisplayView)
```

Comment:
@maicki looks like you're correct. I had seen those comments, but "can be represented" didn't clarify for me that there is no protocol that can be used. However, there does not seem to be a way.

It looks like this is an artifact of the API being design around 3.0, which predates buy a few years the time that Apple started with widely using protocols for many important features such as dynamic animation items.  Based on the functionality of these APIs, it is actually quite surprising they're not supported as a protocol!